core/ecschema-editing/src/ecschema-editing.ts

```diff
@@ -12,6 +12,8 @@ export * from "./Validation/Rules";
 export * from "./Validation/SchemaValidationVisitor";
 export * from "./Validation/SchemaWalker";
 export * from "./Validation/SchemaCompareDiagnostics";
+export * from "./Validation/SchemaChanges";
+export * from "./Validation/SchemaCompareReporter";
```

Comment:
Can we export SchemaComparer module as well?  @christophermlawson

Reply:
Yes, I can add that.